pkg/api/v1/clients/kube/resource_client.go

```diff
@@ -282,7 +294,10 @@ func (rc *ResourceClient) List(namespace string, opts clients.ListOpts) (resourc
 	for _, resourceCrd := range listedResources {
 		resource, err := rc.convertCrdToResource(resourceCrd)
 		if err != nil {
-			return nil, errors.Wrapf(err, "converting output crd")
+			resource, err = rc.convertCrdToInvalidResource(resourceCrd)
```

Comment:
i think we should propagate the conversion error here. set the `reason` field on the Status so the user knows where parsing failed